sct.py

```diff
@@ -1594,7 +1595,23 @@ def generate_parallel_timelines_report(logdir: str | None, test_id: str | None)
     pt_report_generator.generate_full_report()
 
 
-cli.add_command(configure_jenkins_builders)
+@cli.command("configure-jenkins-builders", help="Configure all required jenkins builders for SCT")
+@cloud_provider_option
+@click.option("-r", "--regions", type=CloudRegion(), default=[], help="Cloud regions", multiple=True)
+def configure_jenkins_builders(cloud_provider, regions):
+    add_file_logger()
+    logging.basicConfig(level=logging.INFO)
+
+    match cloud_provider:
+        case 'aws':
+            AwsCiBuilder(AwsRegion('eu-west-1')).configure_auto_scaling_group()
```

Comment:
Don't we want to use first region from the `regions` var instead of hardcoded `eu-west-1`?

Reply:
no, this is the CI builder, and we want them in specific region

Reply:
i.e. this the ASG used for SCT CI